static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js

```diff
@@ -103,13 +105,30 @@ define([
         }
     }
 
+    function paidforCardLoaded(el) {
+        var height,
+            $adSlot = $(el);
+
+        if ($adSlot.hasClass('ad-slot--mostpop')) {
+            fastdom.read(function () {
+                height = $adSlot.dim().height;
+            });
+
+            fastdom.write(function () {
+                $('.js-most-popular-footer .tabs__pane').css('height', height);
```

Comment:
I don't think so. Based on the examples in documentation is seems that `read` will be always before `write`. Now I noticed that I could move `$('.js-most-popular-footer .tabs__pane')` to the `read` operation.